Junior-Devs/2020-Junior-Devs-Project
# Frontend setup

Solves https://github.com/Junior-Devs/2020-Junior-Devs-Project/issues/16

### Test instructions:

- `cd frontend`
- `npm run dev` (make sure you're inside the `frontend` folder)
- Confirm that the 'Hello World' message is displayed and the app works
